components/pipeline-service/staging/stone-stage-p01/resources/tekton-results-s3-secret-path.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 - op: add
   path: /spec/dataFrom/0/extract/key
-  value: integrations-output/terraform-resources/app-sre-stage-01/stone-stage-p01/redhat-stg-plnsvc-s3
+  value: integrations-output/terraform-resources/appsres05ue1/stone-stage-p01/redhat-stg-plnsvc-s3
```

Comment:
It should be `stone-stage-p01-plnsvc-s3` instead of `redhat-stg-plnsvc-s3`

Reply:
done